aten/src/ATen/native/native_functions.yaml

```diff
@@ -3396,8 +3396,13 @@
   variants: function, method
 
 - func: pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: pixel_shuffle
+    QuantizedCPU: quantized_pixel_shuffle_cpu
```

Comment:
nit: pixel_shuffle_quantized_cpu

we have aligned on the naming before to ${func_name}_${backend_name}

Reply:
Updated!